### PR TITLE
Find dat folder.

### PR DESCRIPTION
**Pull request by kozlac**
Made changes to be able to find the dat folder when running from out of source folder. This resolves issue #604

### CODE REVIEW

**Comment by intendo** [comment]
@sobolevnrm I approve the changes made to find the .DAT files (e.g. PARSE.DAT).
Do you want to review the changes and approve the pull request?

**Comment by sobolevnrm** [comment]
@intendo - I reviewed when it showed up in my queue and posted some follow-on questions for the underlying issue #604.  I'm not trying to be difficult but I want to make sure we're "fixing" the right problem.  It's not obvious that this patch is preferable to developing a different method for managing paths in PDB2PQR -- I'd like to make sure we're considering larger changes that will make the code more useful/sustainable in the long run.

**Comment by kozlac** [comment]
Added some new changes so that we can find the /dat folder regardless of where pdb2pqr is being run. Also, I added the setup.py script so that we can run pdb2pqr as a module. I'm excluding pdb2pka from the installation for now. And now that I remember I may be missing some dependencies in the setup script. 

The building and installation are the same as propka:
`$ cd pdb2qqr`
`$ python3 -m pip install --upgrade .`
`$ python3 -m pdb2pqr  [flags|options] <file_name>.pdb <file_name>.pqr`

**Comment by sobolevnrm** [comment]
I will try to review this weekend; sorry for the delay.

File: pdb2pqr/pdb2pqr/main.py [hunk 1]
```diff
@@ -431,9 +435,12 @@ def non_trivial(args, protein, definition, is_cif):
             _LOGGER.info("Assigning titration states with PROPKA.")
             protein.remove_hydrogens()
             pka_df, pka_filename = run_propka(args, protein)
-            _LOGGER.error("\n" + str(pka_df))
-            raise NotImplementedError(
-                "PROPKA not implemented. See %s" % pka_filename)
+
+            protein.apply_pka_values(forcefield_.name, args.ph, dict(zip(pka_df.group_label, pka_df.pKa)));
+
+            # _LOGGER.error("\n" + str(pka_df))
```
Comment (sobolevnrm) [inline pdb2pqr/pdb2pqr/main.py:441]: Can you remove these commented-out lines, please?

File: pdb2pqr/setup.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+MAJOR = 3
+MINOR = 0
+MICRO = 0
+VERSION = "%d.%d.%d" % (MAJOR, MINOR, MICRO)
+
+setuptools.setup(
+    name = "pdb2pqr",
+    version = VERSION,
+    author = "Baker, Nathan A. et al.",
+    author_email = "nathan.baker@pnnl.gov",
```
Comment (sobolevnrm) [inline pdb2pqr/setup.py:36]: Let's use nathanandrewbaker@gmail.com here.

File: pdb2pqr/setup.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    description = "Automates many of the common tasks of preparing structures for continuum solvation calculations as well as many other types of biomolecular structure modeling, analysis, and simulation.",
+    long_description = long_description,
+    install_requires = ["propka >= 3.2", "pandas >= 1.0", "pytest>=5.4.1"],
+    url = " https://github.com/Electrostatics/apbs-pdb2pqr/tree/master/pdb2pqr",
+    packages = setuptools.find_packages(exclude = ["pdb2pka", "*.pdb2pka", "pdb2pka.*", "*.pdb2pka.*"]),
```
Comment (sobolevnrm) [inline pdb2pqr/setup.py:41]: I don't think we should include `pdb2pka` yet -- it's not functional.  Is it needed for this PROPKA-only fix?

File: pdb2pqr/setup.py [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    package_data = {
+        "pdb2pqr": ["dat/*.xml", "dat/*.DAT", "dat/*.names"]
+    },
+    license = "BSD",
+    classifiers = [
+        "Programming Language :: Python :: 3",
```
Comment (sobolevnrm) [inline pdb2pqr/setup.py:47]: We probably want to include 3.5-3.8 explicity since we don't support versions <3.5

**Comment by sobolevnrm** [comment]
@kozlac and @intendo --  do we have test coverage for this PR?

File: pdb2pqr/pdb2pqr/main.py (hunk 1 above)
Comment (sobolevnrm) [reply pdb2pqr/pdb2pqr/main.py:441]: What happened to following the PEP8 style guide?  I just looked at this code and it has both a semicolon and an overly long line.

Comment (sobolevnrm) [reply pdb2pqr/pdb2pqr/main.py]: I fixed this with https://github.com/Electrostatics/apbs-pdb2pqr/pull/605/commits/0ee632b7c49e86276053dce64a8c3fde42c5d8c7

File: pdb2pqr/setup.py (hunk 2 above)
Comment (sobolevnrm) [reply pdb2pqr/setup.py:36]: This code wasn't de-linted either....

Comment (sobolevnrm) [reply pdb2pqr/setup.py]: Fixed in https://github.com/Electrostatics/apbs-pdb2pqr/pull/605/commits/fd4aafc8d390f46f8c49618e95ba0b07364eb4ad

File: pdb2pqr/setup.py (hunk 3 above)
Comment (sobolevnrm) [reply pdb2pqr/setup.py]: Oops -- I read this wrong; it's excluding pdb2pka. Thanks!

File: pdb2pqr/setup.py (hunk 4 above)
Comment (sobolevnrm) [reply pdb2pqr/setup.py]: Fixed in https://github.com/Electrostatics/apbs-pdb2pqr/pull/605/commits/fd4aafc8d390f46f8c49618e95ba0b07364eb4ad

**Comment by sobolevnrm** [comment]
> Added some new changes so that we can find the /dat folder regardless of where pdb2pqr is being run. Also, I added the setup.py script so that we can run pdb2pqr as a module. I'm excluding pdb2pka from the installation for now. And now that I remember I may be missing some dependencies in the setup script.
> 
> The building and installation are the same as propka:
> `$ cd pdb2qqr`
> `$ python3 -m pip install --upgrade .`
> `$ python3 -m pdb2pqr [flags|options] <file_name>.pdb <file_name>.pqr`

Is this documented anywhere?  If not, can we add the documentation?

Note that for development, we probably want to suggest `pip install -e .`.

**Pull request merged**